src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue

```diff
@@ -45,6 +45,7 @@
           class="va-slider__container__handler"
```

Comment:
Selection should be more subtle.
You can check checkbox example

http://localhost:8080/#/demo/vuestic-theme/vuestic-components/va-checkbox/VaCheckbox.demo.vue